compiler/one-cmds/utils.py

```diff
@@ -368,3 +369,32 @@ def _get_optimization_list(get_name=False):
         opt_list = [_remove_suffix(s, '.cfg') for s in opt_list]
 
     return opt_list
+
+
+def _detect_one_import_drivers():
+    """Looks for available import drivers
```

Comment:
I think it'd be good if the comments let us know that it finds import drivers from "current directory".